notebooks/data-sources/github/thoth_PR_data_extraction.ipynb

```diff
@@ -18,7 +18,7 @@
   },
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #2.    data_files = [ospj("../../../../data/", f) for f in os.listdir("../../../../data/")]
</pre><p>Would os.listdir also include directories as well? If that's the case, then I think pd.read_csv could throw an error. Do you wanna add a check that ensures data_files only has csv's? Or alternatively use this instead of os.listdir: <code>glob.glob("./../../../data/*.csv")</code> ?</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/aicoe-aiops/ocp-ci-analysis/pull/498/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='11'/>